coreprofile/10/jakarta-coreprofile-spec-10.0.html

```diff
@@ -0,0 +1,793 @@
+</td>
+</tr>
+</table>
+</div>
+<div class="paragraph">
+<p>As dictated by the general rules for Jakarta EE profiles in the Platform specification, products that implement the Core
+Profile must honor:</p>
+</div>
+<div class="olist arabic">
+<ol class="arabic">
+<li>
+<p>all requirements of the common requirements section that applies to all profiles;</p>
+</li>
+<li>
+<p>all requirements of this specification;</p>
+</li>
+<li>
+<p>all requirements of the individual component specifications;</p>
+</li>
+<li>
+<p>all requirements in the Jakarta EE Platform specification that are conditional on the presence of a specific
+technology or combinations of technologies.</p>
+</li>
+</ol>
+</div>
+<div class="paragraph">
+<p>Let’s look at some examples of requirements from each grouping.</p>
+</div>
+<div class="paragraph">
+<p>For the first one, the Jakarta EE Platform
+specification mandates support for Java&#8482; Platform, Standard Edition 11 API.  Because of this, all Core Profile 10.0 APIs must support the Java&#8482; Platform, Standard Edition 11 API.</p>
+</div>
+<div class="paragraph">
+<p>In the second category one can point out the
+that Jakarta XML Binding support is not required for Jakarta RESTful Web Services in the Core Profile.
+(<a href="#additional_requirements">see Additional Requirements</a>).</p>
+</div>
+<div class="paragraph">
+<p>The third category is hopefully
+self-explanatory. For example, Core Profile products must implement the Jakarta RESTful Web Services API, which in turn means they need to satisfy all the requirements listed in the Jakarta RESTful Web Services 3.1 specification.</p>
+</div>
+<div class="paragraph">
+<p>The fourth category is the most complex. The Jakarta Core Profile TCK attempts to validate the ability to combine the individual component specifications into single test applications.</p>
+</div>
+</div>
+<div class="sect2">
+<h3 id="acknowledgements-for-jakarta-ee-10-0"><a class="anchor" href="#acknowledgements-for-jakarta-ee-10-0"></a>1.3. Acknowledgements for Jakarta EE 10.0</h3>
+<div class="paragraph">
+<p>The Jakarta EE 10.0 specification was created by the Jakarta EE Platform
+Specification Project with guidance provided by the Jakarta EE Working Group
+(<em><a href="https://jakarta.ee/about/working-group/" class="bare">https://jakarta.ee/about/working-group/</a></em>).</p>
+</div>
+</div>
+</div>
+</div>
+<div class="sect1">
+<h2 id="core-profile-definition"><a class="anchor" href="#core-profile-definition"></a>2. Core Profile Definition</h2>
+<div class="sectionbody">
+<div class="paragraph">
+<p>This chapter defines the contents of the Jakarta™ EE 10.0 Core Profile.</p>
+</div>
+<div class="sect2">
+<h3 id="required_components"><a class="anchor" href="#required_components"></a>2.1. Required Components</h3>
+<div class="paragraph">
+<p>The following technologies are required components of the Core Profile:</p>
```

Comment:
I'd sort listed required components alphabetically so it is easier for the reader to find particular spec in the list

Reply:
Ok, reordered alphabetically